contractcourt/channel_arbitrator_test.go

```diff
@@ -1064,6 +1108,121 @@ func TestChannelArbitratorPersistence(t *testing.T) {
 	}
 }
 
+// TestChannelArbitratorForceCloseBreachedChannel tests that the channel
+// arbitrator is able to handle a channel in process of being force close is
```

Comment:
in _the_ process of being force _closed_

Reply:
Fixed: https://github.com/lightningnetwork/lnd/pull/3512/commits/2d09f63e43ede2966fdfeafa25deee1cee9b1a0c